pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -2793,13 +2889,18 @@ def on_reset_clicked(self, *args):
         msg = _("Continuing with this action will reset all your partitioning selections "
                 "to their current on-disk state.")
 
-        dlg = Gtk.MessageDialog(flags=Gtk.DialogFlags.MODAL,
-                                message_type=Gtk.MessageType.WARNING,
-                                buttons=Gtk.ButtonsType.NONE,
-                                message_format=msg)
+        dlg = Gtk.MessageDialog(
+            flags=Gtk.DialogFlags.MODAL,
+            message_type=Gtk.MessageType.WARNING,
+            buttons=Gtk.ButtonsType.NONE,
+            message_format=msg
+        )
         dlg.set_decorated(False)
-        dlg.add_buttons(C_("GUI|Custom Partitioning|Reset Dialog", "_Reset selections"), 0,
-                        C_("GUI|Custom Partitioning|Reset Dialog", "_Preserve current selections"), 1)
+        dlg.add_buttons(
+            C_("GUI|Custom Partitioning|Reset Dialog", "_Reset selections"), 0,
+            C_("GUI|Custom Partitioning|Reset Dialog", "_Preserve current selections"),
+            1
```

Comment:
Please split these numbers to a separate lines to have them the same for both arguments.

Reply:
Fixed.